### PR TITLE
Anna Terelius project 10 survey

### PR DESCRIPTION
**Pull request by AnnaTerelius**
Link to my deployed project:
https://annatereliusproject10survey.netlify.com/

My reflections on how this week's project turned out:
I did mine only for mobile view.

Things I'd like to have clarified or explained in more detail:
I did not manage to set "required" on the input field and dropdown/radiobutton, advice please :)

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,97 @@
...
   return (
-    <div>
-      Find me in src/app.js!
+  <div>{submitted && <Summary name={name} gender={gender} home={home} />}
+    <div className={submitted ? 'formStyleSubmitted' : 'formStyleNotSubmitted'}>
+      {!submitted && ( 
+      <form className="content" onSubmit={event => event.preventDefault()}>
+        <div className="pageTitle">
+          <h1>Home Sweet Home</h1>
+            cat adoption
```
Comment (ericaglimsholt) [inline code/src/App.js]: What element does this belongs to?

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,97 @@
...
+            <input className="formField" 
+              type="email" 
+              placeholder="Example@gmail.com" 
+              onChange={(event)=> setName(event.target.value)} value={name} required/>
+          </label>
+          <label className="newsletter">
+            <input className="checkbox"
+            type="checkbox"
+            checked={wantsNewsletter}
+            onChange={event => setWantsNewsletter(event.target.checked)}/>
```
Comment (ericaglimsholt) [inline code/src/App.js]: Add `required` here to make required

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,97 @@
...
+          <h1>Home Sweet Home</h1>
+            cat adoption
+        </div>
+        <div className="contacts" >
+          <label className="e-mail">
+            <p>e-mail</p>
+            <input className="formField" 
+              type="email" 
+              placeholder="Example@gmail.com" 
+              onChange={(event)=> setName(event.target.value)} value={name} required/>
```
Comment (ericaglimsholt) [inline code/src/App.js]: It's good if you have everything in one line **or** on different rows. So this:
```
<input 
   className="formField" 
   type="email" 
   placeholder="Example@gmail.com" 
   onChange={(event)=> setName(event.target.value)} 
   value={name} 
   required
/>
```
or this:
```
<input className="formField" type="email" placeholder="Example@gmail.com" onChange={(event)=> setName(event.target.value)} value={name} required />
```
don't mix the two

**Comment by AnnaTerelius** [comment]
Hej Erica!
Tack för input.
Hm.. skulle du kunna skriva i kod exakt hur jag ska göra med required, jag provade så som jag tror du menar innan jag skickade in uppgiften (det är därför det är en rest kvar på rad 28..), men det fungerade inte. Det gick då att submitta formuläret utan att ha fyllt i korrekt. Tack på förhand, Anna

Från: Erica Glimsholt <notifications@github.com>
Skickat: den 2 december 2019 22:56
Till: Technigo/project-survey <project-survey@noreply.github.com>
Kopia: AnnaTerelius <tereliusanna@hotmail.com>; Author <author@noreply.github.com>
Ämne: Re: [Technigo/project-survey] Anna Terelius project 10 survey (#19)


@ericaglimsholt approved this pull request.

Hi Anna! I think you did the assignment great. I've added some comments for you to look at. About the required fields. In both <input> and <section> you can add required props to the element, just like it is (like you have done on row 28 in App.js)

________________________________

In code/src/App.js<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-survey%2Fpull%2F19%23discussion_r352877663&data=02%7C01%7C%7Cd5cfac9badc7452dfe7108d777727903%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637109205841797643&sdata=vTDF%2FtTAGrXbkoMZAwQOsURWUy7DJu9vPftkTB9iE%2BI%3D&reserved=0>:

>    return (

-    <div>

-      Find me in src/app.js!

+  <div>{submitted && <Summary name={name} gender={gender} home={home} />}

+    <div className={submitted ? 'formStyleSubmitted' : 'formStyleNotSubmitted'}>

+      {!submitted && (

+      <form className="content" onSubmit={event => event.preventDefault()}>

+        <div className="pageTitle">

+          <h1>Home Sweet Home</h1>

+            cat adoption

What element does this belongs to?

________________________________

In code/src/App.js<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-survey%2Fpull%2F19%23discussion_r352878119&data=02%7C01%7C%7Cd5cfac9badc7452dfe7108d777727903%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637109205841807648&sdata=CDbi2f8i8gVB6SnvQ%2Bw62MsHEfpK3zJG0wP4sqvc6GM%3D&reserved=0>:

> +            <input className="checkbox"

+            type="checkbox"

+            checked={wantsNewsletter}

+            onChange={event => setWantsNewsletter(event.target.checked)}/>

Add required here to make required

________________________________

In code/src/App.js<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-survey%2Fpull%2F19%23discussion_r352878903&data=02%7C01%7C%7Cd5cfac9badc7452dfe7108d777727903%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637109205841817653&sdata=q6M1Wk%2BzQHyOvRaT%2BZ3lXIkoQH1K%2B8PNPwQltj9MfYY%3D&reserved=0>:

> +            <input className="formField"

+              type="email"

+              placeholder="Example@gmail.com"

+              onChange={(event)=> setName(event.target.value)} value={name} required/>

It's good if you have everything in one line or on different rows. So this:

<input

   className="formField"

   type="email"

   placeholder="Example@gmail.com"

   onChange={(event)=> setName(event.target.value)}

   value={name}

   required

/>

or this:

<input className="formField" type="email" placeholder="Example@gmail.com" onChange={(event)=> setName(event.target.value)} value={name} required />

don't mix the two

**Pull request closed without merging**